prow/prowjobs/GoogleCloudPlatform/gcsfuse/gcsfuse-pytorch-dino-periodic.yaml

```diff
@@ -0,0 +1,33 @@
+periodics:
+- name: gcsfuse-pytorch-dino-periodic
+  cluster: gcsfuse-prow-test
```

Comment:
I don't think this is the correct name for the cluster you created. Typically the script creates a cluster with the name build-{TEAM} so I think yours is build-gcsfuse

Reply:
I changed the name in the script to gcsfuse-prow-test and can confirm cluster with this name is present in my project as well.

Reply:
I think maybe you did not edit the script correctly.
It looks like the cluster is configured as build-gcsfuse https://github.com/GoogleCloudPlatform/oss-test-infra/blob/master/prow/oss/gencred-config/gencred-config.yaml

and build-gcsfuse is in a lot of the config https://grep.app/search?q=build-gcsfuse&filter[repo][0]=GoogleCloudPlatform/oss-test-infra

I am not seeing any config for gcsfuse-prow-test

Reply:
okay, I will creating another cluster again.